### ISSUE TITLE
Add entry for R-Ladies Global (blog/news)

### ISSUE THREAD

**Issue by cosimameyer** [rladies/awesome-rladies-creations#58]
I thought I'd open the issue to discuss whether it makes sense to add entries/an entry for the R-Ladies Global [blog](https://www.rladies.org/blog/) and [news](https://www.rladies.org/news/) feed 😊

**Comment by drmowinckels** [comment]
right. In my opinion, I dont think that we should add those two feeds. The idea of blogslist was to have a good way to show everyone how much great work our community members are doing and give them some well-deserved boosts. 

I think our news is for sure not a part of that list, but I could potentially consider adding the blogs, since they are "hopefully" mostly going to be contributed posts by members. I'm not sure.

But its great to have this discussion!
